modules/caddyhttp/reverseproxy/fastcgi/caddyfile.go

```diff
@@ -107,6 +107,12 @@ func (t *Transport) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 				}
 				t.WriteTimeout = caddy.Duration(dur)
 
+			case "capture_stderr":
```

Comment:
Should also add this option to the `php_fastcgi` directive's unmarshal, so it can be used without `transport`

Reply:
whoops, didn't realize the shortcut allowed for overrides, fixed